Scripts/sims4communitylib/utils/misc/common_camera_utils.py

```diff
@@ -104,6 +106,72 @@ def stop_focus_on_object(cls, game_object: GameObject):
         from camera import cancel_focus
         cancel_focus(object=game_object)
 
+    @classmethod
+    def local_start_focus_on_sim(cls, sim_info: SimInfo, follow: bool = True, client: Client = None):
```

Comment:
I don't think we need special "local" functions, it'd make more sense if the original functions attempted to send the operations through omega first, then do it the normal way if it fails to do so.

Reply:
As discussed on Discord, in the context of a regular game both ways of sending the message (through client and through omega) are the same, while in a S4MP setting the difference boils down to whether all players should be focused or only the player for which the command was executed.